magicctapipe/scripts/lst1_magic/semi_automatic_scripts/database_production/set_coord_wobble.py

```diff
@@ -0,0 +1,202 @@
+        if (src in source_dict.keys()) and (source_dict.get(src)[2] != "NaN"):
+            src_point = str(source_dict.get(src)[2])
+            df_LST["point_source"] = np.where(
+                df_LST["source"] == src, src_point, df_LST["point_source"]
+            )
+        else:
+            print(
+                f"\t {i}: {src} extension information not in the dictionaries. Please add it to the dictionaries"
+            )
+            i += 1
+    print("\n\nRetrieving wobble offset...\n\n")
+    df = pd.read_hdf(
+        config_dict["database_paths"]["MAGIC+LST1"],
+        key=config_dict["database_keys"]["MAGIC+LST1"],
+    )  # TODO: put this file in a shared folder
+    if args.begin != 0:
+        df = df[df["DATE"].astype(int) >= args.begin]
+    if args.end != 0:
+        df = df[df["DATE"].astype(int) <= args.end]
+    date_lst = pd.to_datetime(df["DATE"], format="%Y%m%d")
+
+    delta = pd.Timedelta("1 day")
+    date_magic = date_lst + delta
+
+    date_magic = date_magic.dt.strftime("%Y/%m/%d").to_list()
+
+    for i in range(len(df)):
+        magic_runs = (df["MAGIC_runs"].to_list())[i].rstrip("]").lstrip("[").split(", ")
+        lst_run = (df["LST1_run"].to_list())[i]
+        wobble = []
+        source = (df["source"].to_list())[i]
+        for j in range(len(magic_runs)):
+            runs = glob.glob(
+                f"/fefs/onsite/common/MAGIC/data/M[12]/event/Calibrated/{date_magic[i]}/*{magic_runs[j]}*{source}*.root"
+            )
+
+            if len(runs) < 1:
+                print(
+                    f"Neither M1 nor M2 files could be found for {date_magic[i]}, run {magic_runs[j]}, {source}. Check database and stored data!"
+                )
+                continue
+            wobble_run_info = runs[0].split("/")[-1].split(source)[1]
+            if "-W" in wobble_run_info:
+                wobble_run = (wobble_run_info.split("W")[-1])[0:4]
+            else:
+                print(
+                    f"No string matching for wobble offset found in the name of MAGIC files for {date_magic[i]}, run {magic_runs[j]}, {source}. Check it manually!"
+                )
+                continue
+            wobble.append(wobble_run)
+        wobble = np.unique(wobble)
+        if len(wobble) > 1:
+            print(
+                f"More than one wobble offset value for LST run {lst_run}: check data!"
+            )
+        wobble_str = str(wobble).replace(" ", ", ")
+        df_LST["wobble_offset"] = np.where(
+            df_LST["LST1_run"] == lst_run, wobble_str, df_LST["wobble_offset"]
+        )
+
+    df_LST = pd.concat([df_LST, df_LST_full]).drop_duplicates(
+        subset="LST1_run", keep="first"
+    )
+    df_LST = df_LST.sort_values(by=["DATE", "source", "LST1_run"])
+    df_LST = df_LST[df_LST["source"].notna()]
+    df_LST.to_hdf(
+        LST_h5,
+        key=LST_key,
+        mode="w",
+        min_itemsize={
+            "lstchain_versions": 20,
+            "last_lstchain_file": 90,
```

Comment:
note that the new path of DL1 files is something like this:
/fefs/onsite/data/lst-pipe/LSTN-01/DL1/20250704/v0.11/tailcut1005/dl1_LST-1.Run21038.0023.h5
and this has 92 characters, so maybe worth to increase it immediately now to 100 or so?